packages/components/organisms/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue

```diff
@@ -171,8 +170,6 @@ export default {
 </script>
 
 <style lang="scss" module>
-    @include badge();
```

Comment:
what does this remove?

Reply:
I couldn't find the use of this badge styles and they've been deprecated in v6 of fozzie. When running locally there is a warning about it, so I removed it as unused and outdated. Let me know if I missed something and these styles are still needed.

Reply:
At the moment it doesn't add anything, only a warning for the build that this mixin is deprecated

Reply:
cool sounds good to me then.